spec/lib/mvi/configuration_spec.rb

```diff
@@ -34,21 +34,4 @@
       end
     end
   end
-
-  # TODO(knkski): These tests probably aren't doing anything useful.
-  describe '.default_mvi_open_timeout' do
-    context 'when Settings.mvi.open_timeout is not set' do
-      it 'should use the default' do
-        expect(MVI::Configuration::OPEN_TIMEOUT).to eq(2)
-      end
-    end
-  end
-
-  describe '.default_mvi_timeout' do
-    context 'when Settings.mvi.timeout is not set' do
-      it 'should use the default' do
-        expect(MVI::Configuration::TIMEOUT).to eq(10)
-      end
-    end
-  end
```

Comment:
i support removing these default ones, but can we instead replace them to ensure they have whatever is specified in Settings.yml... Ideally, be explicit in the spec.

ie. expect(MVI::Configuration.new.timeout).to eq(15) NOT expect(MVI::Configuration.new.timeout).to eq(Settings.mvi.readtimeout)